sklearn/datasets/descr/twenty_newsgroups.rst

```diff
@@ -219,6 +219,10 @@ Some other classifiers cope better with this harder version of the task. Try
 running :ref:`sphx_glr_auto_examples_model_selection_grid_search_text_feature_extraction.py` with and without
 the ``--filter`` option to compare the results.
 
+.. topic:: Data Considerations
+
+  Cleveland Indians - https://www.mlb.com/news/cleveland-indians-team-name-change 
```

Comment:
It seems that you totally removed the comment

Reply:
@glemaitre , i am sorry. Looks like i misunderstood your comment. I thot you wanted me to have the comments only 80 characters, which is why i removed most of the comment to have only 80 characters. Is it that you want me to add a note below the comment saying "Note-The paragraph is only 80 character long.". Please let me know i shall make the changes.

Reply:
I don't want the change to be less than 80 characters but each line being wrapped in less than 80 characters (similarly to the PEP8 rules).

Reply:
sorry for the misleading comment before.